.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Clean",
+            "type": "shell",
+            "command": "rm -rf /tmp/revel/aaa"
```

Comment:
what is `aaa`?

Reply:
That's the test project created in the launch

Reply:
would `test` be better?